submissions/evgeniy241984/html-css-popup/index.html

```diff
@@ -5,12 +5,169 @@
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="item__link">
+                                <figure class="item__icon icon__plus icon"></figure>
+                            </a>
+                            <span class="item__text">Ooogle+</span>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="item__link">
+                                <figure class="item__icon icon__search icon"></figure>
+                            </a>
+                            <span class="item__text">Search</span>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="item__link">
+                                <figure class="item__icon icon__tube icon"></figure>
+                            </a>
+                            <span class="item__text">Mytube</span>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="item__link">
+                             <figure class="item__icon icon__play icon"></figure>
+                            </a>
+                            <span class="item__text">Play</span>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="item__link">
+                             <figure class="item__icon icon__omail icon"></figure>
+                            </a>
+                            <span class="item__text">Omail</span>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="item__link">
+                                <figure class="item__icon icon__drive icon"></figure>
+                            </a>
+                            <span class="item__text">Drive</span>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="item__link">
+                                <figure class="item__icon icon__calendar icon"></figure>
+                            </a>
+                            <span class="item__text">Calendar</span>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="item__link">
+                                <figure class="item__icon icon__mybusinnes icon"></figure>
+                            </a>
+                            <span class="item__text">My Businnes</span>
+                        </li>
+                    </ul>
+                    <input id="more__btn" type="checkbox" class="more__input">
+                    <label for="more__btn" class="more__label">
+                        <span class="more__btn">More</span>
+                    </label>
+                    <hr class="popup__divider">
```

Comment:
Use CSS for visual elements. Don't litter your markup with unnecessary elements.

Reply:
Do you mean don't use <hr> tag?

Reply:
Yes